<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskTwoWidget</class>
 <widget class="QWidget" name="TaskTwoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1769</width>
    <height>923</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1769</width>
    <height>923</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1769</width>
    <height>923</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>901</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uploadBNFButton">
      <property name="text">
       <string>上传BNF文法文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="grammarAnalyseButton">
      <property name="text">
       <string>文法分析</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>1141</width>
     <height>901</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QTableView" name="firstTable">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>1001</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="followTable">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>490</y>
       <width>1001</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>非终结符的FIRST集合</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>460</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>非终结符的FOLLOW集合</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>页</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>页</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>页</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
